projects/extension/src/components/NetworkTab.tsx

```diff
@@ -27,9 +27,10 @@ export const emojis = {
   seedling: "🌱",
 }
 
-const Accordion = withStyles({
+const Accordion = withStyles((theme) => ({
   root: {
-    border: "1px solid rgba(0, 0, 0, .125)",
+    border: "1px solid",
+    borderColor: theme.palette.divider,
```

Comment:
I think this can be:
```suggestion
    border: {`1px solid ${theme.palette.divider}`},
```